components/create_team/index.js

```diff
@@ -6,17 +6,16 @@ import {connect} from 'react-redux';
 import {getConfig} from 'mattermost-redux/selectors/entities/general';
 import {getCurrentChannel} from 'mattermost-redux/selectors/entities/channels';
 import {getCurrentTeam} from 'mattermost-redux/selectors/entities/teams';
-import {GlobalState} from 'mattermost-redux/types/store';
 
 import CreateTeam from './create_team';
 
-function mapStateToProps(state: GlobalState) {
+function mapStateToProps(state) {
```

Comment:
Why are you changing this file back to JavaScript? If you're running into missing types in `GlobalState`, that import should be switched to import from the webapp's `types/store`

Reply:
There was some strange problem with this file. Looked like a bug in webpack but I didn't investigate further. Changing it back to js fixed the issue. It didn't seem to have anything to do with the file contents.

Reply:
Strange. Well, we can always see later. I doubt this component is one that we're changing enough for autocomplete and such to be really helpful.